editoast/src/views/timetable/stdcm.rs

```diff
@@ -568,6 +650,29 @@ async fn build_work_schedules(
     res
 }
 
+/// Return the list of speed limits that are active at any point in a given time range
+async fn build_temporary_speed_limits(
+    conn: &mut DbConnection,
+    start_date_time: DateTime<Utc>,
+    end_date_time: DateTime<Utc>,
+) -> Result<Vec<LightTemporarySpeedLimit>> {
+    if end_date_time <= start_date_time {
+        return Ok(Vec::new());
+    }
+    let selection_settings: SelectionSettings<TemporarySpeedLimit> = SelectionSettings::new();
+    let applicable_speed_limits: Vec<LightTemporarySpeedLimit> =
+        TemporarySpeedLimit::list(conn, selection_settings)
+            .await?
+            .into_iter()
+            .filter(|speed_limit| {
+                !(end_date_time <= speed_limit.start_date_time.and_utc()
+                    || speed_limit.end_date_time.and_utc() <= start_date_time)
+            })
+            .map(|speed_limit| speed_limit.into())
```

Comment:
If possible, it's usually better to be more explicit. As a developer, reading `.into()` just make me think "into, ok, but into what?".

```suggestion
            .map(LightTemporarySpeedLimit::from)
```

It will also have the advantage to help type inference and so, you won't need `: Vec<LightTemporarySpeedLimit>` I believe.